tests/config-import.js

```diff
@@ -212,6 +212,22 @@ test('config import - presets', async (t) => {
   t.is(config.warnings.length, 0, `no warnings on the file`)
 })
 
+test('config import - translations', async (t) => {
+  const config = await readConfig(defaultConfigPath)
+  for (let { _, value: translationValue } of config.translations()) {
+    t.is(
+      translationValue.schemaName,
+      'translation',
+      `schemaName is 'translation'`
+    )
+    t.ok(
+      translationValue.schemaNameRef === 'presets' || 'fields',
```

Comment:
What I think you meant:

```suggestion
      translationValue.schemaNameRef === 'presets' ||
        translationValue.schemaNameRef === 'fields',
```

As written, this assertion will always succeed. If `translationValue.schemaNameRef === 'presets'`, the assertion will pass. Otherwise, the value of the expression will be `'fields'`, which is always truthy.

Reply:
ooops!